src/Raven.Server/Commercial/LicenseInfo.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using Raven.Server.Utils;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Commercial
+{
+    public class LicenseLimits
+    {
+        public Dictionary<string, int> CoresByNode { get; set; }
```

Comment:
Change this into an object.
We'll want it to be `NodeLicenseDetails` or some such, and likely move more stuff there.

Reply:
done